qp_shotgun/qc_trim/qc_trim.py

```diff
@@ -186,24 +186,11 @@ def _run_commands(qclient, job_id, commands, msg):
     return True, ""
 
 
-def _gzip_file(path):
-    with open(path, "rb") as in_file:
-        gz_path = '%s.gz' % path
-        with gopen(gz_path, "wb") as out_file:
-            out_file.writelines(in_file)
-
-    return gz_path
-
-
 def _per_sample_ainfo(out_dir, samples, fwd_and_rev=False):
     files = []
     missing_files = []
 
-    if fwd_and_rev:
-        suffixes = ['%s_paired_1.fastq', '%s_paired_2.fastq',
-                    '%s_unmatched_1.fastq', '%s_unmatched_2.fastq']
```

Comment:
Atropos doesn't output unmatched read pairs given our parameters -- only need to keep the paired case (and we should probably also rename ... e.g. `sample1_R1_trimmed.fastq.gz`)